### ISSUE TITLE
reverse direction of the supervisor/advisor tree

### ISSUE THREAD

**Issue by TS404** [fnielsen/scholia#1298]
Currently the arrows point from students to their advisors, but this is counterintuitive to read (reverse-chronological).

A better option would be to have the arrows follow the follow the flow of time. Example: https://tinyurl.com/y6tzmfzc

That way the vertical viewing option also shows it like a family tree:

![image](https://user-images.githubusercontent.com/10216013/101724680-46945900-3b03-11eb-8c35-48f25a90590d.png)

**Comment by egonw** [comment]
I agree the three is hard to navigate. @TS404, what about coloring the people by the century they got their PhD in (additionally)?

**Comment by egonw** [comment]
See also https://github.com/fnielsen/scholia/issues/1293

**Comment by fnielsen** [comment]
We should also color the person in question.

**Comment by fnielsen** [comment]
...and get rid of the GAS query.
